cms/templates/manage_users_lib.html

```diff
@@ -72,7 +72,7 @@ <h3 class="title">${_("Grant Access to This Library")}</h3>
 
       <ol class="user-list" id="user-list">
         <div class="ui-loading">
-          <p><span class="spin"><i class="icon fa fa-refresh"></i></span> <span class="copy">Loading</span></p>
+          <p><span class="spin"><span class="icon fa fa-refresh" aria-hidden="true"></span></span> <span class="copy">Loading</span></p>
```

Comment:
This "loading" paradigm is something we have throughout Studio. It's just for the initial loading of the page (usually), and I don't think we should add additional text.